defender/poap/autotasks/mint-poap/index.js

```diff
@@ -1,42 +1,92 @@
+const stackName = 'poap_minter';
```

Comment:
I think it would be beneficial to run lint/format it with prettier for this file

Reply:
Done!